portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,7 +30,22 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println(WELCOME_MESSAGE);
+    ArrayList<String> sampleComments = new ArrayList<String>();
```

Comment:
nit: List<String> for variable type

(always use the interface type unless you require a specific implementation)